integrations/influxdb_connector/influxdb-timestream-connector/src/lib.rs

```diff
@@ -129,17 +133,27 @@ pub async fn lambda_handler(
         .as_bytes();
 
     match handle_body(client, data, &precision).await {
-        // This is the format required for custom Lambda responses
+        // This is the format required for custom Lambda 1.0 responses
         // https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html
-        Ok(_) => Ok(json!({
-            "statusCode": 200,
-            "body": "{\"message\": \"Success\"}",
-            "isBase64Encoded": false,
-            "headers": {
-                "Content-Type": "application/json"
-            },
-            "cookies": []
-        })),
+        Ok(_) => {
+            let mut response = json!({
+                "statusCode": 200,
+                "body": "{\"message\": \"Success\"}",
+                "isBase64Encoded": false,
+                "headers": {
+                    "Content-Type": "application/json"
+                }
+            });
+            // cargo lambda watch expects a Lambda response in 2.0 format.
+            // This means a "cookies" array must be added to the response.
+            // If this "cookies" array is present and the connector is deployed
+            // with synchronous invocation in a stack, users will receive a
+            // 502 error
+            if std::env::var("local_invocation").is_ok() {
+                response["cookies"] = json!([]);
```

Comment:
This is the major change. This allows the format of responses from the Lambda function to satisfy the expected responses when run locally (with `cargo lambda watch` specifically), with synchronous invocation, and with asynchronous invocation.